src/components/Footer/footerData.js

```diff
@@ -0,0 +1,76 @@
+const footerData = [
```

Comment:
This is nice, you have maintained all the footer data at one place. In this way, you can change footer data easily.